src/rpc/governance.cpp

```diff
@@ -1026,9 +1020,10 @@ UniValue gobject(const JSONRPCRequest& request)
     } else if (strCommand == "getcurrentvotes") {
         // GETVOTES FOR SPECIFIC GOVERNANCE OBJECT
         return gobject_getcurrentvotes(request);
+    } else {
+        gobject_help();
+        return UniValue::VNULL; // avoid compiler warnings
```

Comment:
aren't there some `noreturn` attributes in C++ for exactly this? Like `[[ noreturn ]] void myfunc()`? See https://en.cppreference.com/w/cpp/language/attributes/noreturn

Reply:
Not sure I understand... `gobject` does return smth in all other cases, it's only when command doesn't match any of known ones we have this case.

Reply:
I mean that `gobject_help` and friends could have that attribute, making it unnecessary for these dummy returns after calling it.

Reply:
Ahh, I see. Good point 👍 Fixed.